io.catenax.product_stock/1.0.0/ProductStock.ttl

```diff
@@ -0,0 +1,143 @@
+    bamm:dataType :Position.
+:Position a bamm:Entity;
+    bamm:properties ([
+  bamm:property :orderPositionReference;
+  bamm:optional "true"^^xsd:boolean
+] :lastUpdatedOnDateTime :allocatedStocks);
+    bamm:preferredName "Position"@en;
+    bamm:description "The Position can be located at several stocks. A position may be anonymous or may reference an position within an order."@en.
+:orderPositionReference a bamm:Property;
+    bamm:preferredName "Order Position Reference"@en;
+    bamm:description "The order position reference contains information to identify a position within an order."@en;
+    bamm:characteristic :OrderPositionReferenceCharacteristic.
+:lastUpdatedOnDateTime a bamm:Property;
+    bamm:preferredName "Lasted updated on date"@en;
+    bamm:description "Date and time at which the position has been updated by the supplier's systems. Date and time are set according to a timezone."@en;
+    bamm:characteristic :DateTimeStamp;
+    bamm:exampleValue "2023-04-01T14:23:00.66372+01:00"^^xsd:date.
+:allocatedStocks a bamm:Property;
+    bamm:preferredName "Allocated Stocks"@en;
+    bamm:description "Stocks already allocated to the customer. There is one AllocatedStock per supplierStockLocation."@en;
+    bamm:characteristic :AllocatedStockCollection.
+:OrderPositionReferenceCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Order Position Reference Characteristic"@en;
+    bamm:description "The order position reference includes reference numbers for the order and the ID. As these information are meant for the customer, the customer reference numbers are mandatory."@en;
+    bamm:dataType :OrderPositionReference.
+:AllocatedStockCollection a bamm-c:Collection;
+    bamm:preferredName "Allocated Stock Collection"@en;
+    bamm:description "Collection of AllocatedStocks that may relate to different supplierStockLocations."@en;
+    bamm:dataType :AllocatedStock.
+:OrderPositionReference a bamm:Entity;
+    bamm:properties ([
+  bamm:property :supplierOrderId;
+  bamm:optional "true"^^xsd:boolean
+] :customerOrderId :customerOrderPositionId);
+    bamm:preferredName "Reference to Order Position"@en;
+    bamm:description "Encapsulates the references to identify a position within an order."@en.
+:DateTimeStamp a bamm:Characteristic;
+    bamm:preferredName "Date Time Stamp Characteristic"@en;
+    bamm:description "The Date with the the timezone of the involved site."@en;
+    bamm:dataType xsd:date;
+    bamm:see <urn:uri:ISO-8601>.
+:AllocatedStock a bamm:Entity;
+    bamm:properties (:quantityOnAllocatedStock :supplierStockLocationId);
+    bamm:preferredName "Stock allocated to Customer"@en;
```

Comment:
check for consistent upper case writing in all preferredName tags. Sometimes you write lower case in the first character sometimes uppercase. I would opt for headline writing with uppercase writing of important words